dist/index.html

```diff
@@ -0,0 +1,316 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Catalog</title>
+    <link rel="stylesheet" href="/main.48b3db1d.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+  <script src="/main.48b3db1d.js"></script></head>
+  <body>
+    <header class="header">
+      <div class="header__wrapper">
+        <a href="#home" class="header__logo">
+          <img src="/logo.6023b87e.png" alt="MOYO logo">
+        </a>
+
+        <nav class="navigation">
+          <ul class="navigation__list">
+            <li class="navigation__item">
+              <a href="#" class="navigation__link is-active">
+                Apple
+              </a>
+            </li>
+
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">
+                Samsung
+              </a>
+            </li>
+
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">
+                Smartphones
+              </a>
+            </li>
+
+            <li class="navigation__item">
+              <a href="#" class="navigation__link" data-qa="nav-hover">
+                Laptops & Computers
+              </a>
+            </li>
+
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">
+                Gadgets
+              </a>
+            </li>
+
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">
+                Tablets
+              </a>
+            </li>
+            
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">
+                Photo
+              </a>
+            </li>
+            
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="main">
+      <div class="main__wrapper">
+        <div class="card" data-qa="card">
+          <div class="card__wrapper">
+            <div class="card__image-wrapper">
+              <img src="/imac.7817668a.jpeg" alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)" class="card__image">
+            </div>
+            <p class="card__name">APPLE A1419 iMac 27&Prime; Retina 5K&nbsp;Monoblock (MNED2UA/A)</p>
```

Comment:
it looks like heading
use semantic tags
```suggestion
            <h3 class="card__name">APPLE A1419 iMac 27&Prime; Retina 5K&nbsp;Monoblock (MNED2UA/A)</p>
```